bears/documentation/DocGrammarBear.py

```diff
@@ -0,0 +1,110 @@
+import shutil
+
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocBaseClass import (
+    DocBaseClass)
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class DocGrammarBear(DocBaseClass, LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    REQUIREMENTS = {PipRequirement('language-check', '1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Documentation', 'Spelling', 'Grammar'}
+    CAN_FIX = {'Documentation', 'Spelling', 'Grammar'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('java') is None:
+            return 'java is not installed.'
+        else:
+            try:
+                from language_check import LanguageTool, correct
+                LanguageTool
+                correct
+                return True
+            except ImportError:  # pragma: no cover
+                return 'Please install the `language-check` pip package.'
+
+    def process_documentation(self,
+                              parsed,
+                              natural_language,
+                              languagetool_disable_rules):
+        """
+        This fixes the parsed documentation comment.
+
+        :param parsed:
+            Contains parsed documentation comment.
+        :param natural_language:
```

Comment:
Hm, yeah I see that it's take from there. But I'd still the parameter `locale`, that way users immediately know what input we expect.